cdk/util.py

```diff
@@ -44,6 +46,16 @@ def parse_args():
         help="KMS Key arn to encrypt kubernetes secrets, generated if not provided",
         default=None,
     )
+    template_parser.add_argument("--registry-username", help="Quay.io Registry Username", default=None)
+    template_parser.add_argument("--registry-password", help="Quay.io Registry Password", default=None)
+    template_parser.add_argument("--gcr-json-creds", help="GCR Json Credentials", default=None)
+    template_parser.add_argument("--acm-cert-arn", help="ACM Cert ARN", default="__FILL__")
+    template_parser.add_argument(
+        "--hostname", help="Hostname for deployement (ie domino.example.com)", default="__FILL__"
+    )
+    template_parser.add_argument(
+        "--disable-flow-logs", help="Disable monitoring bucket (temporary option)", action="store_true", default=False
```

Comment:
I've encountered some unreliability when destroying the monitoring bucket. It's being fixed elsewhere, but disabling it here for now.